pom.xml

```diff
@@ -264,6 +264,11 @@
         <artifactId>commons-lang3</artifactId>
         <version>3.12.0</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-text</artifactId>
+        <version>1.10.0</version>
+      </dependency>
       <dependency>
```

Comment:
This is a "new" dependency because it's actually overriding a transitive dependency:

```
[INFO] --- maven-dependency-plugin:3.3.0:tree (default-cli) @ keywhiz-api ---
[INFO] com.squareup.keywhiz:keywhiz-api:jar:0.10.2-SNAPSHOT
[INFO] \- io.dropwizard:dropwizard-testing:jar:1.3.29:test
[INFO]    \- io.dropwizard:dropwizard-core:jar:1.3.29:test
[INFO]       \- io.dropwizard:dropwizard-configuration:jar:1.3.29:test
[INFO]          \- org.apache.commons:commons-text:jar:1.9:test
```